ros/lib/host_inventory.py

```diff
@@ -0,0 +1,10 @@
+import requests
+from ros.config import INVENTORY_URL
+
+
+def fetch_host_from_inventory(insights_id, rh_identity):
```

Comment:
```suggestion
def fetch_hosts_from_inventory(insights_id, rh_identity):
```

Reply:
In HBI every host coming through insights-client has the unique `insights_id` therefore, although this API call returns the list it will always have a single item. Do you think we should still rename it to  `fetch_hosts_from_inventory`?

Reply:
Ok so, insights_id would be unique too? Then keeping it the same makes sense. `fetch_host_from_inventory`